src/service.rs

```diff
@@ -384,11 +384,6 @@ where
     async fn sync(self, request: SyncRequest) -> Result<Result<SyncResponse, SyncError>> {
         info(format!("[REQUEST] processing `sync`:  {:?}.", request)).await?;
         let response = self.state.lock().signer.sync(request);
-        task::spawn(async {
-            if self.save().await.is_err() {
-                let _ = warn("unable to save current signer state").await;
-            }
-        });
```

Comment:
Are you sure this was ever working? If so, yeah, not sure what changed. Last time we discussed this code, I thought the problem was the way `save` was being called with `task::spawn`, but it turns out calling `save` at all was causing some kind of error.

Since we are fetching all UTXOs on `manta-wasm-wallet-api` every time we call sync in any case, saving doesn't much improve speed. Signer just recovers from square one on every startup, and it seems to work just fine, though this will get slow fast enough.